SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -436,4 +440,15 @@ private boolean isValidInteger(String strValue) {
       return false;
     }
   }
+
+  public SingularityBounceRequest validateBounceRequest(SingularityBounceRequest defaultBounceRequest) {
```

Comment:
I know this is a nitpick, but all the other validation methods use `check` instead of `validate` in this class.